src/components/ContentHighlights/tests/ContentHighlightsDashboard.test.jsx

```diff
@@ -0,0 +1,43 @@
+import { screen, render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { MemoryRouter } from 'react-router';
+
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { TEST_COURSE_HIGHLIHTS_DATA } from '../data/constants';
+import ContentHighlightsDashboard from '../ContentHighlightsDashboard';
+
+const mockStore = configureMockStore([thunk]);
+
+const initialState = {
+  portalConfiguration: {
+    enterpriseSlug: 'test-enterprise-id',
+  },
+  highlightUUID: 'test-uuid',
+};
+
+const ContentHighlightsDashboardWrapper = (props) => (
+  <MemoryRouter>
```

Comment:
nit: check out the `renderWithRouter` test utility function. an older version of it still exists in this repo actually in `./src/components/test/testUtils.jsx`. It also exists in `@edx/frontend-enterprise-utils` [here](https://github.com/openedx/frontend-enterprise/blob/master/packages/utils/src/test-utils.jsx#L7)